benchmark/sparse/linear_regression/training_benchmark.md

```diff
@@ -0,0 +1,108 @@
+# Linear Regression Benchmark Results 
+
+## Summary
+```
+   Results below show the performance comparison of linear regression with MXNet vs Keras-Tensorflow using sparse tensors
+```                                                   
+
+### Results
+### Training Benchmark
+#### Input samples are sparse feature vectors with 10,000 dimensions
+### Configuration
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.4                                                      |
+| TensorFlow       | v1.11.0                                                     |
+| MXNet-mkl         | v1.3.0   
+
+###### Using 10 epochs
+#### CPU
+##### Speed
+###### Note speed calculated using an average of 5 runs
+| Instance Type | GPUs  | Batch Size  | Keras-MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
+|-----|-----|-----|-----|-----|
+| C5.8XLarge |   0  | 64  | 715.4 sec | 676.6 sec
+| C5.8XLarge |   0  | 128 | 346.2 sec | 341.1 sec 
+| C5.8XLarge |   0  | 256 | 168.9 sec | 165.5
+| C5.8XLarge |   0  | 512 | 89.3 sec | 83.8 sec 
+| C5.8XLarge |   0  | 1024 | 50.5 sec | 48.6 sec
+
+#### Memory consumed
+| Instance Type | GPUs  | Batch Size | Keras-MXNet (CPU Memory (MB)) | Keras-TensorFlow (CPU Memory (MB))  |
+|-----|-----|-----|-----|-----|
+| C5.8XLarge |   0  | 64  | 1331.3 | 1578.2 |
+| C5.8XLarge |   0  | 128 | 1387.3 | 1472.1 
+| C5.8XLarge |   0  | 256 | 1431.3 | 1511.2  |
+| C5.8XLarge |   0  | 512 | 1458.2 | 1506.5 |
+| C5.8XLarge |   0  | 1024 | 1247.8 | 1329.1 |
+
+#### GPU
+### Configuration
+##### Input samples are sparse feature vectors with 10,000 dimensions
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.4                                                      |
+| TensorFlow-GPU   | v1.11.0                                                      |
+| MXNet-cu90mkl    | v1.3.0   
+
+###### Using 10 epochs
+##### Single GPU
+##### Speed
+###### Note speed calculated using an average of 5 runs
+| Instance Type | GPUs  | Batch Size  | Keras-MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
+|-----|-----|-----|-----|-----|
+| P3.8XLarge |   1  | 64  | 50.9 sec | 49.9 sec
+| P3.8XLarge |   1  | 128 | 34.8 sec | 33.5 sec 
+| P3.8XLarge |   1  | 256 | 18.8 sec | 23.3 sec
+| P3.8XLarge |   1  | 512 | 12.4 sec | 19.0 sec
+| P3.8XLarge |   1  | 1024 | 9.1 sec | 18.8 sec
+
+##### Memory utilization
+| Instance Type | GPUs  | Batch Size | Keras-MXNet (GPU Utilization %) | Keras-TensorFlow (GPU Utilization %)  |
```

Comment:
save as above, please remove

Reply:
Updated header name to indicate GPU utilization